src/vs/platform/issue/electron-main/issueMainService.ts

```diff
@@ -254,7 +263,8 @@ export class IssueMainService implements ICommonIssueService {
 							enableWebSQL: false,
 							enableRemoteModule: false,
 							nativeWindowOpen: true,
-							zoomFactor: zoomLevelToZoomFactor(data.zoomLevel)
+							zoomFactor: zoomLevelToZoomFactor(data.zoomLevel),
+							devTools: true
```

Comment:
Why do you call out `devTools` explicitly?

Reply:
added mistakenly from testing, removed